examples/direct_fidelity_estimation.py

```diff
@@ -218,7 +215,7 @@ def _estimate_pauli_traces_clifford(
     pauli_traces: List[PauliTrace] = []
     for dense_pauli_string in dense_pauli_strings:
         # The code below is equivalent to calling
-        # clifford_state.state_vector() and then calling
+        # clifford_tableau.state_vector() and then calling
```

Comment:
Just realized that CliffordTableau doesn't have a state_vector() method (yet). This will need some rephrasing.

Reply:
Done.